src/UI/Component/Input/Container/Factory.php

```diff
@@ -29,4 +29,71 @@
 	 * @return  \ILIAS\UI\Component\Input\Container\Form\Factory
 	 */
 	public function form();
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Filters are used to let the user limit content within a table, list or any other collection of items
+	 *      presented on the screen.
+	 *   composition: >
+	 *      Filters are composed of two visually separated areas:
+	 *      First, there is the Filter Bar at the top. It contains an Expand/Collapse Glyph on the left side
+	 *      On the right, an "Apply" Bulky Button, a "Reset" Bulky Button and a Toggle Button are placed. An additional
+	 *      row with set filter settings will be shown at the bottom of the Filter Bar, when the Filter Bar is collapsed
+	 *      and the Filter is activated at the same time.
+	 *      Second, there is an area where so-called Filter Fields are displayed. Every Filter Field holds a Label on the left
+	 *      and a "Remove" Glyph on the right. Every Filter Field holds an Input Field. After the last Filter Field,
+	 *      an "Add" Bulky Button is shown if additional Filter Fields can be added.
```

Comment:
The lables and remove glyphs are not part of the **Input Fields**. They are attached to "something" that wraps the input fields and occur multiple times within the **Filter**. So the lables and remove glyphs are part of something smaller than the Filter which is not an Input Field. For us it was easier if we give this "something" a name. Otherwise it is hard to address this concept in the description and rules.

Can you suggest an alternative text?

.

Reply:
Something like: 
```
Second, there is an area where the Input Fields of the filter are displayed. The fields are rendered with their label on the left and a "Remove"-Glyph on the right. The values, that are currently entered in the Input fields, are displayed as non-editable text and will get editable when the user focuses an Input Field (see effect).
```
The difference of this is not huge, but talking about how `Input Fields` are rendered in the `Filter` will spare us a term (and hence term, definition, concept, ...).

Reply:
Jour Fixe, 08 OCT 2018 : We would like simply to speak of "Input Fields" (that are rendered in a specific way).